File: auto-docs/executables/r/basic_histogram.r
# Learn about API authentication here: {BASE_URL}/r/getting-started
# Find your api_key here: {BASE_URL}/settings/api

library(plotly)

x = rnorm(500)

py <- plotly(username='TestBot', key='r1neazxo9w')
data <- list(
  list(
    x = x, 
    type = "histogram"
  )
)
response <- py$plotly(data, kwargs=list(filename="basic-histogram", fileopt="overwrite"))
url <- response$url

File: auto-docs/executables/python/basic_histogram.py
# Learn about API authentication here: {{BASE_URL}}/python/getting-started
# Find your api_key here: {{BASE_URL}}/settings/api

import plotly.plotly as py
from plotly.graph_objs import *

import numpy as np
x = np.random.randn(500)

py.sign_in('TestBot', 'r1neazxo9w')
data = Data([
    Histogram(
        x=x
    )
])
plot_url = py.plot(data, filename='basic-histogram', auto_open=False)
